cypress/e2e/admin/administrateCtlTools.feature

```diff
@@ -3,19 +3,22 @@
 @group-C
 @schedule_run
 @pre_check_test
+@prio_0_staging
```

Comment:
why its only for staging (@prio_0_staging)? this feature is exeutable on both on staging and on dev. AFAIK we only need to separate those features which is only doe staging and can not be executed on dev?